src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -88,6 +88,13 @@
 				<div
 					class="navigation">
 					<!-- First page -->
+					<button
+						v-if="page===0 && isPublic"
```

Comment:
```suggestion
						v-if="page === 0"
```

I don't see a reason to limit this. Maybe you want to invite people later or something.

Reply:
I think it would be misleading to have both options there at all times

Reply:
then fix the spacing and lets go for now

Reply:
What´s the problem with the spacing?

Reply:
There should be spaced before and after `===`, but I know see it's all over in this file. Maybe that linting rule is only php 😿

Reply:
Oh I was thinking about spacing in design terms xD